app/src/main/java/de/westnordost/streetcomplete/quests/kerb_height/KerbHeight.kt

```diff
@@ -0,0 +1,9 @@
+package de.westnordost.streetcomplete.quests.kerb_height
+
+enum class KerbHeight(val osmValue: String) {
+    RAISED("raised"),
+    LOWERED("lowered"),
+    FLUSH("flush"),
+    KERB_RAMP("lowered"),
```

Comment:
Wait a moment, @matkoniecz , is this a bug? That both kerb ramp ans lowered have the osm value "lowered"?

Reply:
Not a bug. It is a bit stupid to present fake distinction, but I have not found a better interface.

See https://wiki.openstreetmap.org/wiki/Key:kerb#Tagging 

It was less explicit in the proposal (that never went to vote), but present from start - see https://wiki.openstreetmap.org/wiki/File:FI-Raised-regular-lowered.png from https://wiki.openstreetmap.org/w/index.php?oldid=1207438

Tag to distinguish this would be nice but I will leave it to others. 

I opened https://wiki.openstreetmap.org/wiki/Talk:Key:kerb#Both_kerb_ramps_and_lowered_kerbs_are_tagged_in_the_same_way to increase chance of someone inventing tagging for this.